components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -367,8 +368,8 @@ export class Panel extends React.Component<Props, State> {
 
         if (result === '0') {
           const currency = onlyAnonWallet ? getMessage('bap') : getMessage('bat')
-          const fixed = utils.convertProbiToFixed(notification.args[3])
-          text = getMessage('contributeNotificationSuccess', [fixed, currency])
+          const contributionAmount = new BigNumber(notification.args[3]).toFixed(1, BigNumber.ROUND_DOWN)
```

Comment:
can we just do `notification.args[3].toFixed(1)`?

Reply:
Sure if that's the case - I thought it was a string being passed about. I'll check and change if needed 👍

Reply:
@NejcZdovc Done